charts/akeyless-secure-remote-access/values.yaml

```diff
@@ -255,7 +258,10 @@ ztpConfig:
     memAvgUtil: 50
 
   containerName: "zero-trust-portal"
-  env: []
+  env:
+    # This env variable instruct docker to start only web-bastion processes
+    - name: "BASTION"
```

Comment:
Do we need this for the `portal` as well?

Reply:
no, only `ztb` and `sshProxy`, i udated the code